src/shared/util.js

```diff
@@ -1433,13 +1433,19 @@ MessageHandler.prototype = {
     let targetName = data.sourceName;
     let capability = createPromiseCapability();
 
-    let sendStreamRequest = ({ stream, chunk, success, reason, }) => {
+    let sendStreamRequest = ({ stream, chunk, transfers,
+                               success, reason, }) => {
+      if (transfers) {
```

Comment:
Just thinking, do we really need same changes in [sendStreamResponse](https://github.com/mozilla/pdf.js/blob/master/src/shared/util.js#L1486)? like:

```javascript
let sendStreamResponse = ({ stream, transfers, success, reason, }) => {
  this.postMessage({ sourceName, targetName, stream, 
                     success, streamId, reason, }, transfers);
};
```